docs/antctl.md

```diff
@@ -384,3 +386,32 @@ results:
     componentInfo: Output
     action: Delivered
 ```
+
+### Antctl Proxy
```

Comment:
Would you mention antctl will get the controller/agent Node IP from K8s API, and connects to that IP?

Reply:
Good idea, updated that section